### PR TITLE
Update statuspage incident copy

### PR DESCRIPTION
**Pull request by Aierie**
I've updated incident name to be more focused on the user since this is shown in Card Wallet + the DApp's degraded service notices.

### CODE REVIEW

File: packages/hub/routes/checkly-webhook.ts [hunk 1]
```diff
@@ -38,19 +38,19 @@ export default class ChecklyWebhookRoute {
...
       incidentMessage:
-        'We are experiencing degraded service with our archive RPC node. This will result in reduced transaction processing times.',
+        'We are experiencing degraded service with our archive RPC node. This will result in increased transaction processing times.',
     },
     'xdai non-archive health check - late-cold-smoke (eth_blockNumber)': {
       componentName: 'RPC node (non-archive)',
-      incidentName: 'RPC Node Degradation',
+      incidentName: 'Transactions delayed',
       incidentMessage:
         'We are experiencing degraded service with our non-archive RPC node. This will result in reduced application performance.',
```
Comment (Aierie) [inline packages/hub/routes/checkly-webhook.ts:49]: Unsure about the implications of "reduced application performance" here, outside of transactions. I think the DApp still might try to request some data from the layer 2 chain directly, but for the most part we are getting our data from the subgraph.

Comment (habdelra) [reply packages/hub/routes/checkly-webhook.ts:49]: Yeah, but the subgraph gets its information from the rpc node that the warning is about.

Comment (habdelra) [reply packages/hub/routes/checkly-webhook.ts:49]: As well as the relay server uses the same rpc node the warning is about. Basically when this node goes down, so does the relay server, which means the apps will not be able to send any txns to layer 2

**Review by habdelra — APPROVED** [review]

**Review by backspace — APPROVED** [review]
I appreciate the framing of the language being about the effects for the user, as they’ll often be unaware of the precise implications of particular subsystem downtime 😀

**Pull request merged**